webcompat/templates/list-issue/issuelist-issue.jst

```diff
@@ -26,7 +26,7 @@
 <article class="grid-row">
   <div class="grid-cell issue x3">
     <h1 class="headline-1 js-no-results">Sorry, no results were found.</h1>
-     <p class="issue-labels">
+     <p class="issue-labels js-Issue-label">
```

Comment:
What's the purpose of this class here?

Reply:
this class gives context to the label links. it grabs `data-remotename`, adds the correct path and creates a link. 

though this, we can use the labels as a filter.

Reply:
roger that, thanks.